docs/remote/ssh-tutorial.md

```diff
@@ -71,9 +71,9 @@ You will then be prompted to enter a secure passphrase, but you can leave that b
 
 ## Add SSH key to your VM
 
-In the previous step, you generated an SSH key pair. Take the public key and paste it into your VM setup, by copying the entire contents of the `id_rsa.pub`. You also want to allow your VM to accept inbound SSH traffic by selecting **Allow selected ports** and choosing **SSH** from the **Select inbound ports** dropdown list.
+In the previous step, you generated an SSH key pair. Select **Use existing public key** in the drop down for **SSH public key source** so that you can use the public key you just generated. Take the public key and paste it into your VM setup, by copying the entire contents of the `id_rsa.pub` in the **SSH public key**. You also want to allow your VM to accept inbound SSH traffic by selecting **Allow selected ports** and choosing **SSH (22)** from the **Select inbound ports** dropdown list.
 
-![Add SSH public key to VM](images/ssh-tutorial/add-ssh-public-key.png)
+![Add SSH public key to VM](images/ssh-tutorial/add-ssh-public-key-updated.png)
```

Comment:
You can just commit the new image to the old image filename. Otherwise we end up with lots of obsolete images on the website. We've got source control so the old image is always with us. :-)

Reply:
That's a great point, it'll always be with us :) Updated the new image name to remove `-updated` and removed the obsolete image.